lib/sqlalchemy/sql/compiler.py

```diff
@@ -6102,6 +6102,10 @@ def update_limit_clause(self, update_stmt):
         """Provide a hook for MySQL to add LIMIT to the UPDATE"""
         return None
 
+    def delete_limit_clause(self, delete_stmt):
```

Comment:
yup....looks like we did update_limit_clause the same way, so this is consistent....